pkg/gadgets/trace/tcpdrop/types/types.go

```diff
@@ -20,7 +20,20 @@ import (
 )
 
 type Event struct {
-	eventtypes.Event
+	// Can't use eventtypes.Event because this gadget doesn't support filtering by containers
+
+	// Timestamp in nanoseconds since January 1, 1970 UTC. An int64 is big
+	// enough to represent time between the year 1678 and 2262.
+	Timestamp eventtypes.Time `json:"timestamp,omitempty" column:"timestamp,template:timestamp,stringer"`
+
+	// Type indicates the kind of this event
+	Type eventtypes.EventType `json:"type"`
```

Comment:
That's why I suggested [this](https://github.com/inspektor-gadget/inspektor-gadget/pull/1370#discussion_r1163364281), but I'm not sure the impact.

Reply:
Yeah, that would also solve the issue, but would also mean refactoring most gadgets, right? IMHO, we should wait for gadgettracermanager also using gadgetCtx for CRs (coming soon™) and then remove the event type completely (at least for gadgets where it's only used to transport errors/warnings/debugging) and instead use the logging factility from the gadgetCtx. That would have the additional benefit of messages being filtered depending on the log level on the server already.

In this special case (tcpdrop not being available using CR), we could just go ahead and move all messaging to the gadgetCtx already to not have to worry about the EventType/Message anymore. But if @alban reintroduces `eventtypes.Event`, we don't have to worry about this for now anyway.

Reply:
I agree to delay refactoring until the "CR" work is done. In the meanwhile, I handled the `ErrorGetter` interface manually, my goal is to clean this up to continue with https://github.com/inspektor-gadget/inspektor-gadget/pull/1370, so I'd like to avoid having so many changes on this one if possible. What do you think?